Bottom: 98863da99df53bf5eff6827ef9839f62a03bbc0c
Top:    d4626f107ea0d33715338a8429dbc8f88c3b0f63
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 23:50:11 +0000

Refresh of play-with-perspective-renderin

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 984a0ca..cf76ee7 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1659,8 +1659,6 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   Settings.ShowSolderSide = save_show_solder;
 }
 
-static int one_shot = 1;
-
 #define Z_NEAR 3.0
 gboolean
 ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
